util/test/checkChplInstall

```diff
@@ -96,6 +96,26 @@ else
     CHPL_DIR_REMOVE=0
 fi
 
+# Tmp directory for compile output
+TMP_TEST_DIR="$(mktemp -d ${CHPL_DIR}/chapel-test-XXXXX)"
+log_info "Temporary test job directory: ${TMP_TEST_DIR}"
+
+# Cleanup the tmp directory whenever exit is invoked
+function cleanup_tmp_dir()
+{
+    log_debug "Removing ${TMP_TEST_DIR}"
+    rm -rf ${TMP_TEST_DIR} || (
+        log_warning "failed Removing ${TMP_TEST_DIR}, ignore the error"
+        set -x; ls -la ${TMP_TEST_DIR}
+    ) || : ok
```

Comment:
Post-fixing a shell command with "|| <anything to generate a zero exit status>" means the shell will continue even if the shell command gave an error, as ls -la ${TMP_TEST_DIR} might do; even if set -e  mode is active.  In any case it does no harm. 

"|| : ok" is a recognizable shorthand for the above usage - whenever I see that, I know immediately why it is there.